cmd/mungedocs/unversioned_warning_test.go

```diff
@@ -99,7 +99,7 @@ refer to the docs that go with that version.
 <!-- TAG RELEASE_LINK, added by the munger automatically -->
 <strong>
 The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.1/filename.md).
+[here](http://releases.k8s.io/release-1.2/filename.md).
```

Comment:
I was thinking the same.  We update cmd/mungedocs/mungedocs.go with sed to change the `latestReleaseBranch` and run this entirely heavyweight process to basically do what sed could have done directly on these .md files?  I guess it's doing much more?  
Why isn't `latestReleaseBranch` just taken from a check of the local repo or a call to github?

This is probably discussion for another Issue/PR though.  This change needs to get in, yes?

Is there an 'owner' for cmd/mungedocs at this point?